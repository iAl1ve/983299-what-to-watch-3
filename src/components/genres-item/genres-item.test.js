import React from "react";
import renderer from "react-test-renderer";
import GenresItem from "./genres-item.jsx";

it(`GenresItem renders correctly`, () => {
  const tree = renderer
    .create(
        <GenresItem
          genre={`test`}
          activeClass={`activeItem`} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
