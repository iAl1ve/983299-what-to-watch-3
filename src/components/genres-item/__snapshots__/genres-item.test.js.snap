// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenresItem renders correctly 1`] = `
<li
  className="catalog__genres-item activeItem"
>
  <a
    className="catalog__genres-link"
    href="#"
  >
    test
  </a>
</li>
`;
